en/docs/install-and-setup/install/running-the-mi-in-automation-mode.md

```diff
@@ -9,34 +9,32 @@ WSO2 Micro Integrator runtime offers two operational modes:
+Historically confined to server mode, MI's exclusive operational mode limited its adaptability, particularly in dynamic cloud environments that demanded short-lived, task-specific executions. The newly introduced automation mode addresses these constraints by providing a flexible mechanism for streamlined task execution, aligning seamlessly with cloud-native best practices. In this mode, MI executes the specified automation sequence from start to finish, allowing for efficient task executions without the need for a persistent server process.
 
 ## Before you begin
 
 Before you try out the steps in this guide, complete the following:
 
 - [Download and install]({{base_path}}/install-and-setup/install/installing-mi) the latest Micro Integrator.
-- Download the relevant [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/) based on your operating system.
+- Launch Visual Studio Code with the Micro Integrator extension installed.
+
+    !!! info
+        Follow the [Install Micro Integrator for VS Code]({{base_path}}/develop/mi-for-vscode/install-wso2-mi-for-vscode) documentation for a complete installation guide.
 
 
 ## Design the integration
 
-The main sequence is essential in the automation mode. It serves as the core of your task's integration logic, dictating its execution path. A well-constructed main sequence ensures a consistent and optimal flow for your tasks in the automation mode. Follow the steps below to design the main sequence:
+The automation sequence is essential in the automation mode. It serves as the core of your task's integration logic, dictating its execution path. A well-constructed main sequence ensures a consistent and optimal flow for your tasks in the automation mode. Follow the steps below to design the main sequence:
 
-1. Open WSO2 Integration Studio.
-2. [Create an Integration Project]({{base_path}}/develop/create-integration-project) with **Create ESB Configs** and **Create Composite Exporter** options enabled.
-3. Create a sequence with your integration scenario.
-4. Define the configured sequence as a main sequence for the composite application.
-      <img src="{{base_path}}/assets/img/setup-and-install/select-main-sequence.png" title="Select Main Sequence" width="700" alt="Select Main Sequence"/>
-      <img src="{{base_path}}/assets/img/setup-and-install/configure-main-sequence.png" title="Add Main Sequence" width="600" alt="Add Main Sequence"/>
+1. [Create an Integration Project]({{base_path}}/develop/create-integration-project).
+2. Create a sequence with your integration scenario.
+3. Define the configured sequence as an automation sequence for the composite application. Go to **MI Overview** using the **Home** icon. Select **Set as automation sequence**. 
```

Comment:
Shall we rewrite this step in separate steps?

Also we can add the following steps too.

4. Go to **MI Overview** using the **Home** icon.
5. Under Sequences, locate the sequence you defined.
6. Click **More Actions** (indicated by three dots).
7. Select Set as Automation Mode.